mattermost-plugin/webapp/src/components/rhsChannelBoardItem.test.tsx

```diff
@@ -52,19 +54,29 @@ describe('components/rhsChannelBoardItem', () => {
                 },
             },
             boards: {
+                current: board.id,
+                boards: {
+                    [board.id]: board,
+                },
                 myBoardMemberships: {
                     [board.id]: {userId: 'user_id_1', schemeAdmin: true},
                 },
-                boards: {
-                    [board.id]: board
-                }
-            }
+            },
+            users: {
+                boardUsers: {
+                    user_id_1: {username: 'abc'},
+                },
+            },
+            clientConfig: {
+                value: {},
+            },
```

Comment:
Same here, are these changes necessary? Or only necessary because of changes to "TestBlockFactory"?

Reply:
These changes were required for the MarkdownEditor component. Forgot to remove it. Will remove it. Thanks.